[2022-05-22 22:04:18,186] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: book_db.save_books 2022-05-22T22:03:52.975384+00:00 [queued]>
[2022-05-22 22:04:18,226] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: book_db.save_books 2022-05-22T22:03:52.975384+00:00 [queued]>
[2022-05-22 22:04:18,228] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 22:04:18,229] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2022-05-22 22:04:18,230] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 22:04:18,297] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): save_books> on 2022-05-22T22:03:52.975384+00:00
[2022-05-22 22:04:18,301] {standard_task_runner.py:52} INFO - Started process 125 to run task
[2022-05-22 22:04:18,309] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'book_db', 'save_books', '2022-05-22T22:03:52.975384+00:00', '--job-id', '6', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/books_db.py', '--cfg-path', '/tmp/tmp1gp5_fzq', '--error-file', '/tmp/tmpo31af35x']
[2022-05-22 22:04:18,311] {standard_task_runner.py:77} INFO - Job 6: Subtask save_books
[2022-05-22 22:04:18,565] {logging_mixin.py:104} INFO - Running <TaskInstance: book_db.save_books 2022-05-22T22:03:52.975384+00:00 [running]> on host 929b3ffb0527
[2022-05-22 22:04:18,730] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=book_db
AIRFLOW_CTX_TASK_ID=save_books
AIRFLOW_CTX_EXECUTION_DATE=2022-05-22T22:03:52.975384+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-22T22:03:52.975384+00:00
[2022-05-22 22:04:18,751] {python.py:151} INFO - Done. Returned value was: None
[2022-05-22 22:04:18,763] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=book_db, task_id=save_books, execution_date=20220522T220352, start_date=20220522T220418, end_date=20220522T220418
[2022-05-22 22:04:18,883] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-05-22 22:04:18,937] {local_task_job.py:151} INFO - Task exited with return code 0
